r2i/onnxrt/model.h

```diff
@@ -29,7 +29,6 @@ class Model : public IModel {
  public:
   Model();
 
-  std::shared_ptr<Ort::Session> GetOnnxrtSession();
```

Comment:
What is going on here?

Reply:
This was a leftover from previous PRs, not needed anymore.